library/src/tensor_operation_instance/gpu/pool3d_fwd/device_avg_pool3d_fwd_ndhwc_i8_instance.cpp

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2024, Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
New files should have copyright with current year - the year of creation.
```suggestion
// Copyright (c) 2024, Advanced Micro Devices, Inc. All rights reserved.
```